README.md

```diff
@@ -1,178 +1,302 @@
+### Install it
 
-## About
+```bash
+$ npm install contentful-metalsmith
+```
 
-This plugin for [metalsmith](http://www.metalsmith.io) allows you to build a static site using the data stored at [Contentful](http://www.contentful.com). This
-plugin is built on top of the [Contentful JavaScript Client](https://github.com/contentful/contentful.js).
+### Configure globals
 
-## TL;DR
+When you use metalsmith using the [cli](https://github.com/metalsmith/metalsmith#cli) edit your `metalsmith.json` and add `contentful-metalsmith` in the plugins section.
 
-1. Install it
+```javascript
+// metalsmith.json
 
-  ```javascript
-    $ npm install contentful-metalsmith
-  ```
-2. Configure it (example for [metalsmith CLI](https://github.com/segmentio/metalsmith#cli))
+{
+  "source": "src",
+  "destination": "build",
+  "plugins": {
+    "contentful-metalsmith": {
+      "access_token" : "YOUR_CONTENTFUL_ACCESS_TOKEN"
+    }
+  }
+}
+```
 
-  ```javascript
-    $ vim metalsmith.json
+When you use the [JavaScript Api](https://github.com/metalsmith/metalsmith#api) add `contentful-metalsmith` to the used plugins.
 
-    ----
+```javascript
+metalsmith.source('src')
+metalsmith.destination('build')
 
-    {
-        "source": "src",
-        "destination": "build",
-        "plugins": {
-          ...,
-          "contentful-metalsmith": {
-              "accessToken" : "YOUR_CONTENTFUL_ACCESS_TOKEN"
-          },
-          ...
-        }
-    }
-  ```
-
-3. Create a source file
-
-  ```html
-
-    ---
-    title: OMG metalsmith-contentful
-    contentful:
-        space_id: AN_SPACE_ID
-    template: entries.html
-    ---
-
-    [OPTIONAL CONTENT]
-  ```
-
-4. Create the template (handlebarsjs on this case)
-
-  ```html
-    <!doctype html>
-
-    <html lang="en">
-    <head>
-        <meta charset="utf-8">
-
-        <title>Contentful-metalsmith plugin example</title>
-        <meta name="description" content="No description">
-        <meta name="author" content="Contentful">
-
-    </head>
-
-    <body>
-
-      {{contents}}
-          <table>
-            <thead>
-                <tr>
-                    <th>Type</th>
-                    <th>Id</th>
-                    <th>Updated</th>
-                </tr>
-              </thead>
-              <tbody>
-              {{#each contentful.entries}}
-                <tr>
-                    <td>
-                      {{this.data.sys.contentType.sys.id}}
-                  </td>
-                    <td>
-                      {{this.data.sys.id}}
-                    </td>
-                    <td>
-                      {{this.data.sys.updatedAt}}
-                    </td>
-                </tr>
-                {{/each}}
-              </tbody>
-           </table>
-    </body>
-  </html>
-  ```
-
-5. Enjoy
+metalsmith.use(require('contentful-metalsmith')({ 'access_token' : 'YOUR_CONTENTFUL_ACCESS_TOKEN' }))
+```
 
-# Usage
+**Global parameters:**
```

Comment:
Hmm, I'm not a fan of `:` in headlines. Aren't they already introducing a section?